theatre/studio/src/panels/DetailPanel/propEditors/utils/useTempTransactionEditingTools.tsx

```diff
@@ -0,0 +1,41 @@
+import getStudio from '@theatre/studio/getStudio'
+import type {SerializableValue} from '@theatre/shared/utils/types'
+import type {
+  CommitOrDiscard,
+  ITransactionPrivateApi,
+} from '@theatre/studio/StudioStore/StudioStore'
+import type {IEditingTools} from './IEditingTools'
+import {useMemo} from 'react'
+
+export function useTempTransactionEditingTools<T extends SerializableValue>(
```

Comment:
This wraps tempTransactions? Why did you create this? Add a comment on example usage.